bootstrap.d/x11-libs.yml

```diff
@@ -608,7 +605,7 @@ packages:
         - '--without-doxygen'
         - '--with-sysroot=@SYSROOT_DIR@' # Set libtool's lt_sysroot.
         environ:
-          PYTHON: '@BUILD_ROOT@/tools/host-python/bin/python3.8'
+          PYTHON: '/usr/bin/python3'
```

Comment:
Why is this done? If we already have a host-python, why don't we use it?

Reply:
host-python seems a bit silly, but so is hardcoding this path if it's overriding $PATH

Reply:
Host-python doesn't have xcbgen.

Reply:
`PYTHON: python3` should also do that, no? also, isn't the default py3 now on debian?

Reply:
Seems to work fine, I've changed it.